### PR TITLE
Add missing modules to docs/modules.org

### PR DESCRIPTION
**Pull request by alarsyo**

### CODE REVIEW

File: docs/modules.org [hunk 1]
```diff
@@ -157,6 +160,7 @@ Modules that offer terminal emulation.
 Small modules that give Emacs access to external tools & services.
 
 + ansible - TODO
++ biblio - TODO
```
Comment (hlissner) [inline docs/modules.org:163]: Could you remove biblio from here. While I do intend to merge #2888, it hasn't been merged yet and should be its prerogative to do so.

Comment (alarsyo) [reply docs/modules.org]: Done!

**Comment by alarsyo** [comment]
TODO keywords added!

**Comment by hlissner** [comment]
Excellent work. Thanks for the help!

**Pull request merged**